src/TrackGenerator.cpp

```diff
@@ -747,13 +755,13 @@ void TrackGenerator::initializeTracks() {
 
       /* If Track points to the upper right */
       if (i < _num_azim / 2)
-        _tracks[i][_num_x[i]+j].getStart()->setCoords(0,
-                                     dy_eff[i] * (0.5 + j));
+        _tracks[i][_num_x[i]+j].getStart()->setCoords
+          (0, dy_eff[i] * (0.5 + j), z);
```

Comment:
I think the opening paranthesis is typically put on the preceding rather than the trailing line